### PR TITLE
[dompurify] : Added new ALLOWED_NAMESPACES config

### PR DESCRIPTION
**Pull request by kevin-deyoungster**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test dompurify`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/cure53/DOMPurify/blob/e50e814dfc3117e5a2f3f661819b041c9b25ba1d/src/purify.js#L435-L438
- [x] ~If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.~

### CODE REVIEW

File: types/dompurify/test/dompurify-tests.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ DOMPurify.sanitize(dirty, { FORBID_ATTR: ['style'] }); // $ExpectType string
 DOMPurify.sanitize(dirty, { FORBID_TAGS: ['style'] }); // $ExpectType string
 DOMPurify.sanitize(dirty, { KEEP_CONTENT: false }); // $ExpectType string
 DOMPurify.sanitize(dirty, { NAMESPACE: 'http://www.w3.org/2000/svg' }); // $ExpectType string
+DOMPurify.sanitize(dirty, {ALLOWED_NAMESPACES: ['http://www.w3.org/2000/svg']}); // $ExpectType string
```
Comment (peterblazejewicz) [inline types/dompurify/test/dompurify-tests.ts:28]:
```suggestion
DOMPurify.sanitize(dirty, { ALLOWED_NAMESPACES: ['http://www.w3.org/2000/svg'] }); // $ExpectType string
```
when in doubt, run prettier on your changes:
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
you have to update the version in the `index.d.ts` header:
```ts
// Type definitions for DOM Purify 2.4
```
> If updating a package for new functionality, don't forget to update the version number to line up with that version of the library. If users make sure versions correspond between JavaScript packages and their respective @types packages, then npm update should typically just work.

https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Comment by kevin-deyoungster** [comment]
Thanks for the reviews @peterblazejewicz. I updated the version # to  `2.4.1` to correspond with the current release.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@kevin-deyoungster thanks!

**Comment by kevin-deyoungster** [comment]
Ready to merge

**Pull request merged**